src/app/routes/utils/regex/utils/index.js

```diff
@@ -5,6 +5,7 @@ const legacyAssetUriRegex = '[a-z0-9-_]{1,}/[a-z0-9-_/]{1,}';
 const variantRegex = '/simp|/trad|/cyr|/lat';
 const articleLocalRegex = 'articles|erthyglau|sgeulachdan';
 const mediaIdRegex = '[a-z0-9]+';
+const optionalIdRegex = '/[a-z0-9]+';
```

Comment:
A suggestion on how to insert the conditional / here would be great:

:brandId(${mediaIdRegex}):mediaId(${optionalIdRegex})?

Currently copying the method used for variants

Reply:
Having the `/` here will include it in the `mediaId` - I don't think that value is actually used anywhere, but seems wrong anyway

(variant has this problem at this point too, but it is removed here: https://github.com/bbc/simorgh/blob/706edd65525153496b768b2f0d47e197f0de4fc9/src/app/lib/utilities/variantHandler/index.js#L21)

Reply:
>A suggestion on how to insert the conditional / here would be great
I think these functions can return an array of regexes, or you can pass an array in the route file

ie, one for podcast, one for podcast brands

Reply:
Done!